packages/@react-types/breadcrumbs/src/index.d.ts

```diff
@@ -25,14 +25,29 @@ export interface BreadcrumbItemProps extends PressEvents {
 
 export interface BreadcrumbsProps<T> {
   children: ReactElement<ItemProps<T>> | ReactElement<ItemProps<T>>[],
+  /**
+   * Current number of visible items before collapse.
+   * @default "auto"
+   */
   maxVisibleItems?: 'auto' | number,
   onAction?: (key: Key) => void
 }
 
 export interface SpectrumBreadcrumbsProps<T> extends BreadcrumbsProps<T>, DOMProps, StyleProps {
+  /**
+   * What the Breadcrumbs's size should be.
+   * @default "M"
+   */
   size?: 'S' | 'M' | 'L',
+  /** Whether the last item should have role of "heading". */
   isHeading?: boolean,
+  /**
+   * Sets the aria-level attribute of the last item, but only if `isHeading` is true.
+   * @default 1
```

Comment:
is this default `1`? it kind of is, but only if `isHeading` is set. i can remove -- someone lmk :)

Reply:
yes, default is 1 if isHeading is set to true